boards/BMS/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -65,105 +65,142 @@ void App_SetPeriodicSignals_AccumulatorInRangeChecks(
 {
     App_Accumulator_ReadCellVoltages(accumulator);
 
-    struct InRangeCheck *min_cell_voltage_in_range_check =
```

Comment:
Don't know what I was doing here. The functions to get the in-range check pointers are now called in `App_SetPeriodicSignals_InRangeCheck()`